easybuild/easyconfigs/t/Trinity/Trinity-2.6.5-foss-2017a.eb

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
@timeu @ebirn Style nitpicking: please remove this empty line

Reply:
will do.